Tangem/Common/UI/DefaultListViews/DefaultRowView/DefaultRowView.swift

```diff
@@ -49,6 +48,21 @@ struct DefaultRowView: View {
         .contentShape(Rectangle())
     }
 
+    private var titleView: some View {
+        HStack(spacing: 1) {
+            Text(viewModel.title)
+                .style(appearance.font, color: appearance.textColor)
+            if let secondaryAction = viewModel.secondaryAction {
+                Button(action: secondaryAction) {
+                    Image(systemName: "info.circle")
+                        .frame(width: 13)
+                        .padding(5)
+                        .foregroundStyle(Colors.Icon.informative)
```

Comment:
```suggestion
                        .foregroundColor(Colors.Icon.informative)
```
Color же обычно, он deprecated, но все равно пока его используем